registry.json

```diff
@@ -1,4 +1,19 @@
 [
+    {
+        "ticker": "RELAY",                                                      
+        "poolId": "6c1c1fc2b1e3e1ac056f6dd4dd4c3b78e66bd01896ca6e98a4d2431e",
+        "social": {                                                            
```

Comment:
Please can you add some kind of social presence? The website you have listed doesn't seem to be finished and the reddit account has very few posts / content.

Reply:
Is it not normal for people to register with the alliance upon registering their pool on the mainnet to show their support for decentralization? Do most pools have a social following even before they register as a pool? I'm curious because I saw a couple of these requests for changes after submitting my pull request.

My pool was just registered on the mainnet and I'm setting up a social presence before marketing and advertising (unless you count the exposure to other SPOs when asking for help and helping in forums and alliances I am a member of). I must have thought of it as a "gotta have somewhere to send people before you start sending them" kinda thing.

I went ahead and canceled the pull request for now, but really would like to know if a lack of any current social clout denies entry into this alliance and if so, also what kind of numbers would suffice for entry into the metadata.

Reply:
Most pools have at least some social media presence once their pools are registered or shortly after. We just ask that there is some evidence of reaching out to delegators. The minimum I usually accept is a twitter account with a few posts and some followers / following. It's not a big ask and doesnt take a lot of effort. 

@leo42 does that seem fair?

Reply:
That seems reasonable. Joining an alliance will usually get that much traction within a few hours on Twitter, and there are plenty of alliances without this no-bootstrappers requirement. I would think most pools consider joining alliances as one of the very first forms of attracting or reaching out to delegators, though I understand some OG alliance members may feel it dilutes their own marketing impact in this regard.

You may want to include this information in the README to avoid surprising people with change requests. There are many hurdles to be overcome for mass adoption, but this seems like one straw that doesn't really need to be on the camel's back. I do understand the quality over quantity patient faith in growth, though. It's what brought me to the Cardano camp.

While it may seem antithetical at first to do otherwise, does this preclude private pools from showing their support for the "one operator / one pool" model by joining the group? While it would give them no marketing boost (no delegators to market to), I belive it would help the network overall by showing a more united front in what at first appears to be an alliance primarily supporting decentralization.

Also, while I get the allure of semi-passive income that seems to bring many pools to the space, some people probably run pools for more than personal financial gain. I have a belief in the Cardano blockchain that will keep my public nodes running for decades even if my public pool picks up no delegators. i.e. The pool doesn't cost me more to run than I am willing to invest for a better future even if I find no time to manipulate potential investors into climbing onboard.